rules/windows/defense_evasion_msbuild_beacon_sequence.toml

```diff
@@ -23,8 +23,8 @@ type = "eql"
 
 query = '''
 sequence by process.entity_id
-  [process where event.type in ("start", "process_started") and process.name == "MSBuild.exe"]
-  [network where process.name == "MSBuild.exe" and
+  [process where event.type in ("start", "process_started") and process.name : "MSBuild.exe"]
+  [network where process.name : "MSBuild.exe" and
      not (destination.address == "127.0.0.1" and source.address == "127.0.0.1")]
```

Comment:
```suggestion
     not (destination.ip == "127.0.0.1" and source.ip == "127.0.0.1")]
```

Reply:
address -> ip can be accounted for here or in https://github.com/elastic/detection-rules/pull/400